static/gsApp/views/seerAutomation/seerAutomationProjectList.tsx

```diff
@@ -49,8 +57,10 @@ function ProjectSeerSetting({project, orgSlug}: {orgSlug: string; project: Proje
 
 export function SeerAutomationProjectList() {
   const organization = useOrganization();
-  const {projects, fetching, fetchError} = useProjects();
+  const api = useApi();
+  const {projects, fetching, fetchError, reloadProjects} = useProjects();
   const [page, setPage] = useState(1);
+  const [selected, setSelected] = useState<Set<string>>(new Set());
```

Comment:
callback style runs it once when created
```suggestion
  const [selected, setSelected] = useState<Set<string>>(() => new Set());
```